src/reviews/README.md

```diff
@@ -17,46 +17,60 @@ There are several time periods during the whole Call for Proposals and Proposal
 6. Internal Decision
 7. Announcement of Acceptance
 
-Each time period has a set of settings that need to be set in src/pycontw2016/settings, or
+Each time period has a set of settings that need to be set in the Django administration dashboard, or
 some steps you might need to go through, which will be documented here.
 
 ### 1. Call for Proposals
 
-`settings.py`:
+In the administration dashboard, go to `Home > Django Registry > Entries` with corresponding `SLUG`.
+`SLUG` is the corresponding string of the conference evet, e.g. `pycontw-2020` for 2020.
+For example, if you are running the version of year 2020,
+you may also access the console via `https://<host address>:8000/2020/admin/registry/entry/`
```

Comment:
I think you don't need the `2020` in the development environment, can you check it with your own environment @tai271828

Reply:
Confirmed: no `2020` in the development environment.

Reply:
Updated the corresponding session. Thanks!